Cannot build on windows - not a valid Win32 application

When building on Windows, during the maven `convert-sql-scripts` execution, the following error occurs:

`
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:exec (convert-sql-scripts) on project owaspSecurityShepherd: Command execution failed. Cannot run program "C:\Users\user\git\SecurityShepherd\docker\scripts\convert-sql-scripts.sh" (in directory "C:\Users\user\git\SecurityShepherd"): CreateProcess error=193, %1 is not a valid Win32 application -> [Help 1]
`

This is due to windows being unable to directly run a script from maven, even with a bash environment present. It could be solved by making bash the executable and passing the script as an argument.

--- a/pom.xml
+++ b/pom.xml
@@ -179,7 +179,10 @@
 									<goal>exec</goal>
 								</goals>
 								<configuration>
-									<executable>${basedir}/docker/scripts/convert-sql-scripts.sh</executable>
+									<executable>bash</executable>
+									<arguments>
+										<argument>${basedir}/docker/scripts/convert-sql-scripts.sh</argument>
+									</arguments>
 								</configuration>
 							</execution>
 						</executions>

